### PR TITLE
Site updates

### PR DESCRIPTION
**Pull request by ianyimi**
kiras site updates as well as a couple other customer request updates

### CODE REVIEW

File: src/pages/balloonski.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+const LinkTree: NextPage = () => {
+  return (
+    <>
+      <Head>
+        <title>{ARTIST.name} | Muse Place</title>
+      </Head>
+      <Standard
+        scrollData={scrollData}
+        socials={ARTIST.socialLinks}
+        audio="https://d27rt3a60hh1lx.cloudfront.net/audio/ini-bestmixever.mp3"
```
Comment (alex-shortt) [inline src/pages/balloonski.tsx:54]: remove audio parameter, default is set in context now

File: src/pages/djupgrade.tsx [hunk 2]
```diff
@@ -2,6 +2,7 @@ import { NextPage } from "next";
 import Head from "next/head";
 import dynamic from "next/dynamic";
 import { GothamProps } from "../themes/Gotham";
+import { Text } from "spacesvr";
```
Comment (alex-shortt) [inline src/pages/djupgrade.tsx:5]: are you using this?

File: src/pages/measme.tsx [hunk 3]
```diff
@@ -1,17 +1,16 @@
 import { NextPage } from "next";
 import Head from "next/head";
 import dynamic from "next/dynamic";
-import Gotham, { GothamProps } from "../themes/Gotham";
-const Codame = dynamic(import("../scenes/Gotham/Codame"), { ssr: false });
+import { GothamProps } from "../themes/Gotham";
+const Gotham = dynamic(import("../scenes/Gotham"), { ssr: false });
```
Comment (alex-shortt) [inline src/pages/measme.tsx:5]: don't use `..`

File: src/pages/rebekahkeida.tsx [hunk 4]
```diff
@@ -0,0 +1,57 @@
+import { NextPage } from "next";
+import Head from "next/head";
+import dynamic from "next/dynamic";
+import { ScrollData } from "../themes/Alto/types/scroll";
```
Comment (alex-shortt) [inline src/pages/rebekahkeida.tsx:4]: dont need `..`

File: src/scenes/Gotham/index.tsx [hunk 5]
```diff
@@ -12,6 +12,7 @@ export type GothamSceneProps = {
   night?: boolean;
   children?: ReactNode;
   audio?: string;
+  coupon?: string;
```
Comment (alex-shortt) [inline src/scenes/Gotham/index.tsx:15]: is this being used?

Comment (alex-shortt) [reply src/scenes/Gotham/index.tsx:15]: oh i see where it's being used. the theme has its own props as opposed to the scene, this should be placed on the theme

**Review by alex-shortt — CHANGES_REQUESTED** [review]

File: src/scenes/Gotham/index.tsx (hunk 5 above)
Comment (ianyimi) [reply src/scenes/Gotham/index.tsx:15]: yeah It is on the theme. I first tried it on just the theme alone and it should have worked but it wasn't going through for some reason I was so confused

Comment (alex-shortt) [reply src/scenes/Gotham/index.tsx:15]: double checked it and yeah it should be working without this line... let me know if it still isn't

**Review by alex-shortt — APPROVED** [review]

**Pull request merged**